### PR TITLE
Update 2.8 release notes

### PR DESCRIPTION
**Pull request by kasiakepka**
**Description**

Changes proposed in this pull request:
- add information about CNI breaking changes

**Related issue(s)**
[Customer Incident](https://support.wdf.sap.corp/sap/support/message/2270174384&saprole=ZCSSINTPROCE)

### CODE REVIEW

File: content/blog-posts/2022-10-21-release-notes-2.8/index.md [hunk 1]
```diff
@@ -94,9 +94,16 @@ For more details on the changes, read the official [Istio 1.15.0 release notes](
 
 ### Istio CNI plugin
 
-This Kyma version introduces the Istio CNI plugin. The plugin replaces the `istio-init` container, and it provides the same networking functionality, but it doesn't require Istio users to have elevated Kubernetes RBAC permission. 
+**Breaking Changes**
+This Kyma version introduces the Istio CNI plugin. The plugin replaces the `istio-init` container, and it provides the same networking functionality, but it doesn't require Istio users to have elevated Kubernetes RBAC permission. To learn more, read about the [Istio CNI plugin](https://istio.io/latest/docs/setup/additional-setup/cni/).
 
-To learn more, read about the [Istio CNI plugin](https://istio.io/latest/docs/setup/additional-setup/cni/).
+To avoid issues it is necessary to follow one of the below migration guides:
```
Comment (barchw) [inline content/blog-posts/2022-10-21-release-notes-2.8/index.md:100]:
```suggestion
If init-containers starting in a Pod with sidecar injection enabled need to have networking capabilities, it is necessary to follow one of the below migration guides: 
```

File: content/blog-posts/2022-10-21-release-notes-2.8/index.md [hunk 2]
```diff
@@ -94,9 +94,16 @@ For more details on the changes, read the official [Istio 1.15.0 release notes](
...
+This Kyma version introduces the Istio CNI plugin. The plugin replaces the `istio-init` container, and it provides the same networking functionality, but it doesn't require Istio users to have elevated Kubernetes RBAC permission. To learn more, read about the [Istio CNI plugin](https://istio.io/latest/docs/setup/additional-setup/cni/).
 
-To learn more, read about the [Istio CNI plugin](https://istio.io/latest/docs/setup/additional-setup/cni/).
+To avoid issues it is necessary to follow one of the below migration guides: 
+ 
+1. Set the `uid` of the init container to `1337` using `runAsUser`. `1337` is the `uid` used by the sidecar proxy. Traffic sent by this `uid` is not captured by the Istio's iptables rule. Application container traffic will still be captured as usual. 
+
+2. Set the `traffic.sidecar.istio.io/excludeOutboundIPRanges` annotation to `disable` it disables redirecting traffic to any CIDRs the `init containers` communicate with. 
+
+3. Set the `traffic.sidecar.istio.io/excludeOutboundPorts` annotation to `disable` it disables redirecting traffic to the specific outbound ports the `init containers` use.
```
Comment (IwonaLanger) [inline content/blog-posts/2022-10-21-release-notes-2.8/index.md:106]:
```suggestion
- Set the UID of the `init` container to `1337` using `runAsUser`. `1337` is the UID used by the sidecar proxy. The traffic sent by this UID is not captured by the Istio's iptables rule. Application container traffic is captured as usual. 

- Set the `traffic.sidecar.istio.io/excludeOutboundIPRanges` annotation to `disable`. It disables redirecting traffic to any CIDRs that the `init` containers communicate with. 

- Set the `traffic.sidecar.istio.io/excludeOutboundPorts` annotation to `disable`. It disables redirecting traffic to the specific outbound ports that the `init` containers use. 
```

**Review by barchw — CHANGES_REQUESTED** [review]

File: content/blog-posts/2022-10-21-release-notes-2.8/index.md [hunk 3]
```diff
@@ -94,15 +94,25 @@ For more details on the changes, read the official [Istio 1.15.0 release notes](
 
 ### Istio CNI plugin
 
-This Kyma version introduces the Istio CNI plugin. The plugin replaces the `istio-init` container, and it provides the same networking functionality, but it doesn't require Istio users to have elevated Kubernetes RBAC permission. 
-
+**Breaking news**
```
Comment (IwonaLanger) [inline content/blog-posts/2022-10-21-release-notes-2.8/index.md:97]:
```suggestion
**Breaking changes**  
```

File: content/blog-posts/2022-10-21-release-notes-2.8/index.md [hunk 4]
```diff
@@ -94,15 +94,25 @@ For more details on the changes, read the official [Istio 1.15.0 release notes](
...
 
-This Kyma version introduces the Istio CNI plugin. The plugin replaces the `istio-init` container, and it provides the same networking functionality, but it doesn't require Istio users to have elevated Kubernetes RBAC permission. 
-
+**Breaking news**  
+	
+This Kyma version introduces the Istio CNI plugin. The plugin replaces the `istio-init` container, and it provides the same networking functionality, but it doesn't require Istio users to have elevated Kubernetes RBAC permission.  
+	
 To learn more, read about the [Istio CNI plugin](https://istio.io/latest/docs/setup/additional-setup/cni/).
 
+If init-containers starting in a Pod with sidecar injection enabled need to have networking capabilities, you must follow one of the below migration guides:
```
Comment (IwonaLanger) [inline content/blog-posts/2022-10-21-release-notes-2.8/index.md:103]:
```suggestion
If `init` containers starting in a Pod with sidecar injection enabled need to have networking capabilities, you must follow one of the these migration guides:
```

Comment (barchw) [reply content/blog-posts/2022-10-21-release-notes-2.8/index.md:103]: `init-containers` is the type of the resource, and shouldn't be separated

**Review by IwonaLanger — APPROVED** [review]

File: content/blog-posts/2022-10-21-release-notes-2.8/index.md [hunk 5]
```diff
@@ -94,15 +94,25 @@ For more details on the changes, read the official [Istio 1.15.0 release notes](
...
 
-This Kyma version introduces the Istio CNI plugin. The plugin replaces the `istio-init` container, and it provides the same networking functionality, but it doesn't require Istio users to have elevated Kubernetes RBAC permission. 
-
+**Breaking changes**  
+	
+This Kyma version introduces the Istio CNI plugin. The plugin replaces the `istio-init` container, and it provides the same networking functionality, but it doesn't require Istio users to have elevated Kubernetes RBAC permission.  
+	
 To learn more, read about the [Istio CNI plugin](https://istio.io/latest/docs/setup/additional-setup/cni/).
 
+If `init` containers starting in a Pod with sidecar injection enabled need to have networking capabilities, you must follow one of the these migration guides:
```
Comment (barchw) [inline content/blog-posts/2022-10-21-release-notes-2.8/index.md:103]:
```suggestion
If `init-containers` starting in a Pod with sidecar injection enabled need to have networking capabilities, you must follow one of the these migration guides:
```

**Review by barchw — CHANGES_REQUESTED** [review]

**Review by barchw — DISMISSED** [review]

File: content/blog-posts/2022-10-21-release-notes-2.8/index.md [hunk 6]
```diff
@@ -94,15 +94,25 @@ For more details on the changes, read the official [Istio 1.15.0 release notes](
...
-
+**Breaking changes**  
+	
+This Kyma version introduces the Istio CNI plugin. The plugin replaces the `istio-init` container, and it provides the same networking functionality, but it doesn't require Istio users to have elevated Kubernetes RBAC permission.  
+	
 To learn more, read about the [Istio CNI plugin](https://istio.io/latest/docs/setup/additional-setup/cni/).
 
+If `initContainers` starting in a Pod with sidecar injection enabled need to have networking capabilities, you must follow one of the these migration guides:
+ 
+- Set the UID of the `init` container to `1337` using `runAsUser`. `1337` is the UID used by the sidecar proxy. The traffic sent by this UID is not captured by the Istio's iptables rule. Application container traffic is captured as usual.
```
Comment (IwonaLanger) [inline content/blog-posts/2022-10-21-release-notes-2.8/index.md:105]:
```suggestion
- Set the UID of the `initContainer` to `1337` using `runAsUser`. `1337` is the UID used by the sidecar proxy. The traffic sent by this UID is not captured by the Istio's iptables rule. Application container traffic is captured as usual. 
```

File: content/blog-posts/2022-10-21-release-notes-2.8/index.md [hunk 7]
```diff
@@ -94,15 +94,25 @@ For more details on the changes, read the official [Istio 1.15.0 release notes](
...
+	
 To learn more, read about the [Istio CNI plugin](https://istio.io/latest/docs/setup/additional-setup/cni/).
 
+If `initContainers` starting in a Pod with sidecar injection enabled need to have networking capabilities, you must follow one of the these migration guides:
+ 
+- Set the UID of the `init` container to `1337` using `runAsUser`. `1337` is the UID used by the sidecar proxy. The traffic sent by this UID is not captured by the Istio's iptables rule. Application container traffic is captured as usual. 
+	
+- Set the `traffic.sidecar.istio.io/excludeOutboundIPRanges` annotation to `disable`. It disables redirecting traffic to any CIDRs that the `init` containers communicate with. 
+	
+- Set the `traffic.sidecar.istio.io/excludeOutboundPorts` annotation to `disable`. It disables redirecting traffic to the specific outbound ports that the `init` containers use.
```
Comment (IwonaLanger) [inline content/blog-posts/2022-10-21-release-notes-2.8/index.md:109]:
```suggestion
- Set the `traffic.sidecar.istio.io/excludeOutboundPorts` annotation to `disable`. It disables redirecting traffic to the specific outbound ports that the `initContainers` use. 
```

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**